files/scripts/supervisor-proc-exit-listener

```diff
@@ -179,6 +179,10 @@ def main(argv):
         if len(file_descriptor_list) > 0:
             line = file_descriptor_list[0].readline()
             headers = childutils.get_headers(line)
+            # Check if 'len' exists before using it
+            if 'len' not in headers:
+                syslog.syslog(syslog.LOG_ERR, "WARNING: Missing 'len' header in: {}".format(headers))
```

Comment:
>headers [](http://example.com/codeflow?start=92&length=7)

Could you also print `line` in this message for future troubleshooting purpose?

Reply:
Good suggestion, update code, please review it again, thanks.